asset-transfer-basic/chaincode-go/assetTransfer.go

```diff
@@ -11,6 +11,196 @@ import (
 	"github.com/hyperledger/fabric-samples/asset-transfer-basic/chaincode-go/chaincode"
 )
 
+// SmartContract provides functions for managing an Asset
+type SmartContract struct {
+	contractapi.Contract
+}
+
+// Asset describes basic details of what makes up a simple asset
+type Asset struct {
```

Comment:
this whole chaincode got moved to https://github.com/hyperledger/fabric-samples/blob/59acacbef1c2cfbbc05500bb88535ea6d8f2bebe/asset-transfer-basic/chaincode-go/chaincode/smartcontract.go#L16